ItalianTranslation_PlanetCoaster.xml

```diff
@@ -5760,8 +5760,8 @@
 		<entry key="FrontEndMenu_Player" translation="Sei tu!"/>
 		<entry key="FrontEndMenu_PromptNo" translation="No"/>
 		<entry key="FrontEndMenu_PromptYes" translation="Sì"/>
-		<entry key="FrontEndMenu_Rating" translation="Rating"/>
-		<entry key="FrontEndMenu_RepositionHQ" translation="Modifica HQ Avatar"/>
+		<entry key="FrontEndMenu_Rating" translation="Valutazione"/>
+		<entry key="FrontEndMenu_RepositionHQ" translation="Modifica Avatar su Quartier Generale "/>
```

Comment:
Qui non si capisce, modifichi il quartier generale non l'avatar (come sembra), cambia tipo "Modifica quartier generale avatar"

Reply:
Si, in realtà sarebbe "Riposiziona Avatar su QG". Perchè sposta materialmente l'omino, ma se vuoi va bene anche come hai scritto.